plugins/module_utils/checkpoint.py

```diff
@@ -532,8 +532,11 @@ def api_command(module, command):
 
     code, response = send_request(connection, version, command, payload)
     result = {"changed": True}
+    
+    if command.startswith("show"):
+        result['changed'] = False
 
-    if code == 200:
+    if code == 200 and "incidents" not in response:
```

Comment:
Which module returned a status code of 200 and incidents in its response?

Reply:
This is an API issue since the status code shouldn't be 200 if there were incidents, please open a support ticket.

Reply:
I will open a support ticket and this one change can be excluded.

Reply:
Once you remove this change i will merge this pull request